doc/contributing.md

```diff
@@ -1,20 +1,25 @@
 # How to Contribute
 
+## License
+
 SkyPortal is released under the Modified BSD license, which means that you are allowed to modify the code for your own purposes, as long as you retain our [copyright notice](https://github.com/skyportal/skyportal/blob/master/LICENSE.txt).
 
 However, we would love to grow the SkyPortal community, and integrate improvements directly into our [code repository on GitHub](https://github.com/skyportal/skyportal).
 
-## Code Changes
-
-To make a code contribution to the project:
+## Including your changes
 
-1. Clone the SkyPortal repository
```

Comment:
I've typically forked repositories when I contribute, would this be a good thing to add to step 1?

Reply:
Yes, thanks! Good catch.